pyinfraformat/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .plots import *
 
 logger = logging.getLogger("pyinfraformat")
+logger.propagate = False
 
```

Comment:
Why this?

Reply:
With jupyter "%autoreload 2" didnt work correctly. But maybe we should assume program is used without autoreload. Removed this.